starter-code/css/main.css

```diff
@@ -6,3 +6,95 @@ Green: #00B172
 White: #FFF
 
 */
+
+  /* ---- reset ---- */
+
+  html {
+    box-sizing: border-box;
+  }
+  *, *:before, *:after {
+    box-sizing: inherit;
+  }
+  
+  
+  
+  
+  /* ---- typography ---- */
+  
+  body {
+    font-family: 'Montserrat', sans-serif;
+    width: 100%;
```

Comment:
width is not typography.
You have an ugly thin margin in your body. Try to apply a reset in it (padding and margin = 0);